### ISSUE TITLE
doc: update IntelSGX

### ISSUE THREAD

**Issue by maveraw** [enarx/website#127]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Currently, the information in the IntelSGX system seems inadequate. In order for beginners to have a better understanding, we will add more information to the documentation. Hence making it more undrstandable.

### Suggestions for a technical implementation

Will use some pictorial representation for a better understanding.

**Comment by rjzak** [comment]
Agreed, the SGX set-up is more involved and complicated. There are instructions under the Clouds section here https://enarx.dev/docs/Clouds#other-software. But maybe there should be a generic SGX instruction.
